packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -60,24 +60,34 @@ let scheduleWork: (
   callback: FrameCallbackType,
   options?: {timeout: number},
 ) => number;
-let cancelScheduledWork: (callbackID: number) => void;
+let cancelScheduledWork: (callbackId: number) => void;
 
 if (!ExecutionEnvironment.canUseDOM) {
+  let callbackIdCounter = 0;
+  // Timeouts are objects in Node.
+  // For consistency, we'll use numbers in the public API anyway.
+  const timeoutIds: {[number]: TimeoutID} = {};
```

Comment:
This is not super efficient (number keys are slow) but it can only fire in Node so it probably doesn't matter.
This is a naïve shim anyway.